core/new-gui/src/app/common/service/notification/notification.service.ts

```diff
@@ -4,7 +4,7 @@ import { Observable, Subject } from "rxjs";
 export interface Notification {
   type: "success" | "info" | "error" | "warning" | "loading";
   message: string;
-  duration: number;
+  options: object;
```

Comment:
can you specify the type instead of "object"?

Reply:
The type used in the `NzMessageService` is object. Please check https://ng.ant.design/components/message/en#api

Reply:
let's refer back to their code: https://github.com/NG-ZORRO/ng-zorro-antd/blob/eb2cb046e122eab654a2e721b51f397d2790019e/components/message/typings.ts#L11

Reply:
Replaced `object` with `NzMessageDataOptions`.